### PR TITLE
Requests testing update

### PR DESCRIPTION
**Pull request by perryr16**
## What's this PR do?
* Update Request tests
## Where should the reviewer start? 
* request test
## How should this be manually tested?
* pytest
## Any background context you want to provide?
* n/a
## What are the relevant tickets?
* n/a

### CODE REVIEW

File: src/tests/test_campaign.py [hunk 1]
```diff
@@ -41,14 +41,9 @@ def setUp(self):
 
     with self.app.app_context():
       # create all db objects
+      db.drop_all()
       db.create_all()
```
Comment (perryr16) [inline src/tests/test_campaign.py:45]: `drop_all()` is the same as finding campaigns by name and then deleting them

File: src/tests/test_request.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        'approvals': 0,
+
+    }
+
+
+    with self.app.app_context():
+      # create all db objects
+      db.drop_all()
+      db.create_all()
+      res = self.client().post('/api/v1/campaigns/',
```
Comment (perryr16) [inline src/tests/test_request.py:79]: create 2 campaigns in setup

File: src/tests/test_campaign.py (hunk 1 above)
Comment (AntonioJacksonII) [reply src/tests/test_campaign.py:45]: Nice!

**Comment by AntonioJacksonII** [comment]
Can you .gitignore all of the pycache files and push up again?  
`*.pyc`

**Review by AntonioJacksonII — APPROVED** [review]

**Review by takeller — APPROVED** [review]

**Pull request merged**